cmd/dfdaemon/app/init.go

```diff
@@ -0,0 +1,161 @@
+package app
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"os/user"
+	"path/filepath"
+	"runtime"
+	"syscall"
+	"time"
+
+	"github.com/dragonflyoss/Dragonfly/common/dflog"
+	dferr "github.com/dragonflyoss/Dragonfly/common/errors"
+	"github.com/dragonflyoss/Dragonfly/common/util"
+	"github.com/dragonflyoss/Dragonfly/dfdaemon/config"
+	"github.com/dragonflyoss/Dragonfly/dfdaemon/constant"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+// initDfdaemon sets up running environment for dfdaemon according to the given config
+func initDfdaemon(cfg config.Properties) error {
+	if cfg.Verbose {
+		logrus.Infoln("use verbose logging")
+		logrus.SetLevel(logrus.DebugLevel)
```

Comment:
In fact, no debug logs will be printed when I use `dfdaemon --verbose=true`. Please check it. THX>

Reply:
I suppose this is caused by calling `dflog.InitLog(false, ...)` in `initLogger` after this. I'll fix that.

Reply:
Fixed